docs/content/docs/tutorials/rds-example.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Deploy PostgreSQL + MariaDB Instances Using the ACK RDS Controller"
+description: "Create managed PostgreSQL or MariaDB instances in Amazon Relational Database Service (RDS) from a Amazon Elastic Kubernetes Service (EKS) deployment."
+lead: "Create and use PostgreSQL or MariaDB instances in Amazon RDS using Amazon Elastic Kubernetes Service (EKS)."
+draft: false
+menu:
+  docs:
+    parent: "tutorials"
+weight: 42
+toc: true
+---
+
+The ACK service controller for Amazon Relational Database Service (RDS) kets you manage RDS database instances directly from Kubernetes. This includes the following database engines:
+
+* [Amazon Aurora](https://aws.amazon.com/rds/aurora/) (MySQL & PostgreSQL)
+* [Amazon RDS for PostgreSQL](https://aws.amazon.com/rds/postgresql/)
+* [Amazon RDS for MySQL](https://aws.amazon.com/rds/mysql/)
+* [Amazon RDS for MariaDB](https://aws.amazon.com/rds/mariadb/)
+* [Amazon RDS for Oracle](https://aws.amazon.com/rds/oracle/)
+* [Amazon RDS for SQL Server](https://aws.amazon.com/rds/sqlserver/)
+
+This guide will show you how to create and connect to several types of database engines available in [Amazon RDS](https://aws.amazon.com/rds/) through Kubernetes.
+
+## Setup
+
+Although it is not necessary to use Amazon Elastic Kubernetes Service (Amazon EKS) with ACK, this guide assumes that you have access to an Amazon EKS cluster. If this is your first time creating an Amazon EKS cluster, see [Amazon EKS Setup](https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/deep-learning-containers-eks-setup.html). For automated cluster creation using `eksctl`, see [Getting started with Amazon EKS - `eksctl`](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) and create your cluster with Amazon EC2 Linux managed nodes.
+
+### Prerequisites
+
+This guide assumes that you have:
+
+- Created an EKS cluster with Kubernetes version 1.16 or higher.
+- AWS IAM permissions to create roles and attach policies to roles.
+- Installed the following tools on the client machine used to access your Kubernetes cluster:
+  - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv1.html) - A command line tool for interacting with AWS services.
+  - [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html) - A command line tool for working with Kubernetes clusters.
+  - [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) - A command line tool for working with EKS clusters.
+  - [Helm 3.7+](https://helm.sh/docs/intro/install/) - A tool for installing and managing Kubernetes applications.
+
+### Install the ACK service controller for RDS
+
+You can deploy the ACK service controller for Amazon RDS using the [rds-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/rds-chart). You can download it to your workspace using the following command:
+
+```bash
+helm pull oci://public.ecr.aws/aws-controllers-k8s/rds-chart --version=v0.0.14
```

Comment:
We just released `v0.0.15` with a bug fix :)

Reply:
Excellent. I updated the tutorial. I'll also update my local deployment 😄